packages/lit-dev-content/site/docs/components/lifecycle.md

```diff
@@ -357,22 +357,57 @@ window.onunhandledrejection = function(e) {
 
 ### Implementing additional customization {#reactive-update-cycle-customizing}
 
-#### performUpdate()  {#performupdate}
+This section covers some less-common methods for customizing the update cycle.
 
-Implements the reactive update cycle, calling the other methods, like `shouldUpdate()`, `update()`, and `updated()`.
+#### scheduleUpdate() {#scheduleupdate}
 
-Call `performUpdate()` to immediately process a pending update. This should generally not be needed, but it can be done in rare cases when you need to update synchronously.
+Override `scheduleUpdate()` to customize the timing of the update. `scheduleUpdate()` is called when an update is about to be performed, and by default it calls `performUpdate()` immediately. Override it to defer the update. 
 
-Implement `performUpdate()` to customize the timing of the update cycle. This can be useful for implementing custom scheduling. Note, if `performUpdate()` returns a Promise, the `updateComplete` Promise will await it.
+For example, the following code schedules the update to occur after the next frame paints, which can reduce jank if the update is expensive:
+
+{% switchable-sample %}
+
+```ts
+protected override async scheduleUpdate(): Promise<void> {
+  await new Promise((resolve) => setTimeout(resolve));
+  this.performUpdate();
+}
+```
 
 ```js
-async performUpdate() {
-   await new Promise((resolve) => setTimeout(() => resolve()));
-   return super.performUpdate();
+async scheduleUpdate() {
+  await new Promise((resolve) => setTimeout(resolve));
+  this.performUpdate();
 }
 ```
 
-In this example, the update is performed after paint. This technique can be used to unblock the main rendering/event thread. See the Chrome Dev Summit talk by Justin Fagnani [The Virtue of Laziness](https://www.youtube.com/watch?v=ypPRdtjGooc) for an extended discussion.
+{% endswitchable-sample %}
+
+If you override `scheduleUpdate()`, it's your responsibility to call `performUpdate()`.
```

Comment:
I think a more robust recommendation is to return `super.scheduleUpdate()` (which just does `this.performUpdate()`) because that would work in mixins too, or if we ever made the default scheduleUpdate do more.

So:

```js
async scheduleUpdate() {
  await new Promise((resolve) => setTimeout(resolve));
  return super.scheduleUpdate();
}
```

Reply:
I think it was @kevinpschaaf who suggested that we use `performUpdate()` directly because it was clearer. I could go either way.

Reply:
> because that would work in mixins too

Hmm, yeah I guess. I just think it's really unintuitive semantically that `super.scheduleUpdate()` _does_ the update.

Reply:
I dunno... I usually think that when you override a method, by default you should call `super.thatMethod()` as it does the correct base behavior.

Maybe this case is just different? It's not a huge deal either way.

Reply:
I'm inclined to agree with @justinfagnani but similarly don't think it's a huge deal either way =)

Reply:
SG. Changed back to `super.scheduleUpdate()`.